README.markdown

```diff
@@ -110,7 +110,7 @@ You can also use the official plugin published to the Buf Registry.
 ```yaml
 version: v1
 plugins:
-  - remote: buf.build/stephenh/plugins/ts-proto
+  - plugin: buf.build/community/stephenh-ts-proto
```

Comment:
Newer versions of the `buf` CLI will use the `plugin` key for both local and remote plugins.